src/main/main.ts

```diff
@@ -253,20 +258,31 @@ async function initializeApp() {
     webEnable(win.webContents);
     createTray(path.join(app.getAppPath(), logoImage));
 
-    const u = await checkForUpdates(standalone);
-    if (u && !isV2) update(u, updateOptions);
-    else if (u && isV2)
-      ipcMain.handle("start update", async () => {
-        await update(u, updateOptions);
-      });
+    const context = await checkUpdateRequired(
```

Comment:
Why is it called `context` when it contains the update itself? the name `update` should suffice, I think.

Reply:
`update` is also exists functions in `src/update/update` I wanted naming are different
And `checkUpdateRequired` doesn't contain update, It only checks update required and returns apv info

Reply:
Context means the situation within, and calling it context implies it contains related but not direct information about the update. I'm unsure what name would be great, though; maybe rename the `update` function to `performUpdate` or something? But also I think having the same name for type and function is not very harmful.